src/status_im/subs/wallet/wallet.cljs

```diff
@@ -496,11 +496,13 @@
  :<- [:profile/currency]
  :<- [:profile/currency-symbol]
  (fn [[{:keys [color]} tokens currency currency-symbol]]
-   (mapv #(utils/calculate-token-value {:token           %
-                                        :color           color
-                                        :currency        currency
-                                        :currency-symbol currency-symbol})
-         tokens)))
+   (let [calculated-tokens (mapv #(utils/calculate-token-value {:token           %
```

Comment:
We have two similar pieces of code here. I'd suggest moving them to a separate function, wdyt? 
Also a test can be created for that function.

Reply:
@vkjr i am not sure which piece of code are you referring to?

Reply:
btw, seems we don't need this to be a vector, so our lazy `map` should be enough

Reply:
Thanks @vkjr for the suggestion, refactored the duplicated part to a seperate utility function and wrote a test for it